puppet/modules/web/manifests/vhost/yum.pp

```diff
@@ -27,11 +28,26 @@
     },
   ]
 
+  secure_ssh::receiver_setup { 'yum':
```

Comment:
In other vhosts we've had to create a boolean for this since the type always creates the keys in `/etc/puppet` which CI doesn't have permission to. Ideally we should use Puppet's settings for the state dir (which are dynamic and can use the homedir), but that's also a migration.